curves/bn254/src/lib.rs

```diff
@@ -15,6 +15,7 @@ mod consts {
 
     /// Input length for the multiplication operation.
     pub const ALT_BN128_MULTIPLICATION_INPUT_LEN: usize = 128;
+    pub const ALT_BN128_MULTIPLICATION_INPUT_LEN_FIX: usize = 96;
```

Comment:
```suggestion
    pub const ALT_BN128_MULTIPLICATION_INPUT_LEN: usize = 96;
```
Instead of defining a new public constant, let's just update the existing variable. It is a public constant, but in this case, this should be okay. Then, let's just hardcode the value 128 below.